docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+services:
+  launchpad:
+    build: .
+    container_name: launchpad-dev
+    network_mode: host
+    environment:
+      - KAFKA_BOOTSTRAP_SERVERS=${KAFKA_BOOTSTRAP_SERVERS:-localhost:9092}
+      - LAUNCHPAD_HOST=127.0.0.1
+      - LAUNCHPAD_PORT=2218
+      - PYTHONUNBUFFERED=1
+    volumes:
+      # Mount source code for live reloading during development
+      - ./src:/app/src
+      - ./tests:/app/tests
+    command:
+      - launchpad
+      - devserver
+      - --verbose
+
+  # Optional local Kafka for standalone development (when not using devservices)
+  kafka:
+    image: confluentinc/cp-kafka:7.4.0
+    hostname: kafka
+    container_name: launchpad-kafka
+    profiles: ["standalone"]
+    depends_on:
+      - zookeeper
+    ports:
+      - "9092:9092"
```

Comment:
Probably should limit port exposure to just localhost here and below to lower risk exposure